RELEASE_NOTES.md

```diff
@@ -2,7 +2,150 @@
 
 ## 2.1.15 "Endora" - December 2, 2020
```

Comment:
Update this date to today's date.

Reply:
Doh! I'm sorry, I shelved it and forgot.